controls.py

```diff
@@ -0,0 +1,110 @@
+    def movement_scaler(self, raw, adjustment):
+        if abs(raw) <= self.adjustment:
+            return 0
+        else:
+            if raw > 0:
+                return (raw-self.adjustment)/1.0
+            elif raw < 0:
+                return  (raw+self.adjustment)/1.0       
+                
+    #Creates variables that store the motor labels for the different motors used. Some motor values have also been put in for adjustments that may be made to add the claw          
+    central_motor = 1
+    right_motor = 2
+    left_motor = 3
+    up_quantity = 500
+    right_quantity = 500
+    left_quantity = 500
+    
+    #Correctly encodes the strings that are used to send quantities in order to send them to serial
+    def printer(information):
+        ser.write(information.encode('latin'))
+        
+    #Moves the robot upward by putting the central motor on higher output
+    def move_up(quantity):
+        printer("" + central_motor + "\n" + quantity + "\n")
+        
+    #Moves the robot leftward by putting the left motor on higher output
+    def move_left(quantity):
+        printer("" + left_motor + "\n" + quantity + "\n")
+        
+    #Moves the robot rightward by putting the right motor on higher output
+    def move_right(quantity):
+        printer("" + right_motor +  "\n" + quantity + "\n")
+        
+    #Moves the robot to the front by putting the right motor and the left motor on higher output
+    def move_front(quantity):
+        printer("" + right_motor + "\n" + quantity + "\n")
+        printer("" + left_motor + "\n" + quantity + "\n")
+    #Turns the robot in some direction with the power output provided    
+    def move_turn(quantity):
+        printer("" + right_motor + "\n" + quantity + "\n")
+        printer("" + left_motor + "\n" + -1*quantity + "\n")
+        
+    #Starts controls and sets up event responses to control movements on the controller
+    def controls_move(self):
+        self.running = True
+        while running:
+            claw_movement = 99
+            for event in pygame.event.get():
+                if event.type == pygame.QUIT:
+                    running = False
+                #The buttons are used to control the leftward, rightward, and other motions of the claw. 
+                #These have not been considered yet since the claw and its motor have not been mounted. 
+                #I took the code for the claws out since we are not using those buttons yet
+                #The joy axes are used to control the motion of the robot forward, leftward, rightward, and to allow the robot to turn as well. 
+                #The axis labelled 0 is used for left to right motion and is located on the left joystick. 
+                #The axis labelled 4 is used for up to right motion and is located on the right joystick.
+                elif event.type == pygame.JOYAXISMOTION:
+                    input = event.value
+                    scaled_input = movement_scaler(input, 0.2)
```

Comment:
You can condense lines `89` and `90`. There doesn't appear to be a difference in value between `right_quantity` and `scaled_input` after line 89.